### ISSUE TITLE
Is each epoch using novel OOD data?

### ISSUE THREAD

**Issue by dlmacedo** [hendrycks/outlier-exposure#17]
I guess that, even using seed, during the program using the following line generates new OOD from the 80 million available:

train_loader_out.dataset.offset = np.random.randint(len(train_loader_out.dataset))

I guess this may be overestimenting results as we may be using much more OOD data than originally thought.
